tox.ini

```
@@ -216,6 +226,7 @@ commands_post =
       ); \
       print("codecov-flags=MyPy", file=gh_output_fd); \
       gh_output_fd.close()'
+  #
```

@webknjaz could you please specify what's try to achieve on in command below?

```suggestion
  # Add all available reports to GHA Job Summary
```
Is it corresponds?

---

```suggestion
  # Publish available MyPy-produced text and JSON reports wrapped as Markdown code blocks, to a GHA job summary
```